Agents/Xamarin.Interactive/CodeAnalysis/Evaluating/EvaluationAssemblyContext.cs

```diff
@@ -71,26 +71,30 @@ public void Dispose ()
         Assembly HandleAssemblyResolve (object sender, ResolveEventArgs args)
         {
             Log.Verbose (TAG, $"Handling assembly resolve event for {args.Name}.");
-            Log.Verbose (TAG, $".NET Assembly map contents: ");
-            foreach (var key in netAssemblyMap.Keys)
-                Log.Verbose (TAG, $"    {key}");
 
             Assembly netAssembly;
+            Log.Verbose (TAG, $"Looking up {args.Name} in compiled assembly map.");
             if (netAssemblyMap.TryGetValue (new AssemblyName (args.Name), out netAssembly))
                 return netAssembly;
 
             AssemblyDefinition assembly;
-            if (assemblyMap.TryGetValue (new AssemblyName (args.Name).Name, out assembly)) {
+            var shortName = new AssemblyName (args.Name).Name;
+
+            Log.Verbose (TAG, $"Looking up {shortName} in assembly definition map.");
+            if (assemblyMap.TryGetValue (shortName, out assembly)) {
                 if (args.RequestingAssembly?.ReflectionOnly == true) {
                     if (File.Exists (assembly.Content.Location))
                         return Assembly.ReflectionOnlyLoadFrom (assembly.Content.Location);
 
                     if (assembly.Content.PEImage != null)
                         return Assembly.ReflectionOnlyLoad (assembly.Content.PEImage);
 
-                    throw new Exception (
-                        $"Could not reflection-only load assembly {args.Name}, location " +
+                    Log.Verbose (
```

Comment:
This is changed from an exception because this exception frequently gets eaten up anyway by the assembly load exception that is thrown if/when we return null. This way, we're actually more likely to see this.